modules/analysis/src/service/mod.rs

```diff
@@ -217,31 +237,24 @@ impl AnalysisService {
     fn query_graphs<'a, F>(
         &self,
         query: impl Into<GraphQuery<'a>> + Debug,
-        distinct_sbom_ids: Vec<String>,
+        graphs: Vec<(String, Arc<PackageGraph>)>,
```

Comment:
It looks like it's not necessary to pass a `Vec`, so maybe use `&[Arc<PackageGraph>]` instead.